docs/standard/security/fips-compliance.md

```diff
@@ -0,0 +1,29 @@
+---
+title: FIPS compliance - .NET Core
+description: Explains .NET Core Federal Information Processing Standard (FIPS) compliance.
+ms.date: 11/20/2019
+author: "Rick-Anderson"
+ms.author: "riande"
+---
+
+# .NET Core Federal Information Processing Standard (FIPS) compliance
+
+.NET Core:
+
+* Does **not** implement any cryptographic primitives.
+* Passes cryptographic primitives calls through to the standard modules the underlying operating system provides.
+* Does **not** enforce FIPS compliance in .NET Core apps.
+* Does **not** stop developers from using non-FIPS compliant algorithms, even on systems configured for FIPS compliance.
+
+The system administrator is responsible for configuring the FIPS compliance for an operating system.
+
+If code is written for a FIPS-compliant environment, the developer is responsible for ensuring that non-compliant FIPS algorithms aren't used.
+
+* Non-compliant FIPS algorithms are not used.
+
+For more information on FIPS compliance, see the following articles:
+
+* [Windows FIPS Compliance](/windows/security/threat-protection/fips-140-validation)
+* [Configuring Windows for FIPS Compliance](/windows/security/threat-protection/security-policy-settings/system-cryptography-use-fips-compliant-algorithms-for-encryption-hashing-and-signing)
+* [OpenSSL FIPS Compliance](https://www.openssl.org/docs/fips.html)
+* [Configuring OpenSSL for FIPS Compliance](https://www.openssl.org/docs/fips/UserGuide.pdf)
```

Comment:
I think there's a RHEL-specific one that we might want to link to.  The openssl.org documents are only for one variety of OpenSSL-FIPS.

Reply:
> I think there's a RHEL-specific one that we might want to link to. 
Can you provide a link?

Reply:
https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/6/html/security_guide/sect-security_guide-federal_standards_and_regulations-federal_information_processing_standard ?

Reply:
Yep, that one.